streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBSegmentedBytesStoreTest.java

```diff
@@ -414,8 +417,11 @@ public void shouldLogAndMeasureExpiredRecords() {
         LogCaptureAppender.setClassLoggerToDebug(RocksDBSegmentedBytesStore.class);
         final LogCaptureAppender appender = LogCaptureAppender.createAndRegister();
 
-        context.setStreamTime(Math.max(retention, segmentInterval) * 2);
-        bytesStore.put(serializeKey(new Windowed<>("a", windows[0])), serializeValue(5));
+        bytesStore.put(serializeKey(new Windowed<>("dummy", nextSegmentWindow)), serializeValue(0));
```

Comment:
Is this to "set" a specific `obserservedTime` -- might need a comment

Reply:
yes, it's to set the stream time to a high enough value that it expires the previous segment... I'll add a comment.